styx-scheduler-service/src/main/java/com/spotify/styx/flyte/FlyteAdminClientRunner.java

```diff
@@ -165,7 +165,9 @@ public String createExecution(final RunState runState, final String execName, fi
         // just to keep compatibility with removing dangling executions
         // TODO: base dangling logic on STYX_WORKFLOW_ID, STYX_PARAMETER and STYX_EXECUTION_ID
         //   and the remove these annotations
-        .put(STYX_WORKFLOW_INSTANCE_ANNOTATION, styxVariables.get(STYX_WORKFLOW_ID) + "#" + styxVariables.get(STYX_PARAMETER))
+        .put(STYX_WORKFLOW_INSTANCE_ANNOTATION, styxVariables.get(STYX_COMPONENT_ID) + "#" +
```

Comment:
Lets better change this code as it is done in [KubernetesDockerRunner](https://github.com/spotify/styx/blob/master/styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesDockerRunner.java#L277)

In line 155 we write `runState.workflowInstance()` me can extract it to a variable and then call `workflowInstance.toKey()` here